pkg/webreg/webreg.go

```diff
@@ -1880,37 +1879,310 @@ as the component.
 const releasePage = `
 <h2>The <code>openshift/release</code> repository</h2>
```

Comment:
Could we update this to `"Contributing CI Configurations to the openshift/release Respository"` or somethign

Reply:
Sure. I also added a short blurb to separate the H2 and H3 header at the top, and made the top menu say "Contributing to openshift/release" rather than just "openshift/release".